sdk/iOS/OoyalaSkinSDK/OOUpNextManager.m

```diff
@@ -48,10 +64,10 @@ - (void)setNextVideo:(NSDictionary *)nextVideo {
 
     // After the a new video has been set, let react know that isDismissed
     // is now false.
-    [self.ooBridge sendDeviceEventWithName:@"upNextDismissed" body:@{@"upNextDismissed" : [NSNumber numberWithBool:[self isDismissed]]}];
+    [self.ooReactSkinModel sendEventWithName:upNexDismissedKey body: @{upNexDismissedKey: @(self.isDismissed)}];
```

Comment:
Is this misspelled? upNexDismissedKey should be upNextDismissedKey

Reply:
Can you please change it regardless, for consistency with the rest of the code.

Reply:
Done